.snippets/code/vs-ethereum/tx-fees-block-1901.md

```diff
@@ -5,7 +5,7 @@ const axios = require("axios");
 // according to the transaction type (For Ethereum API: legacy, EIP-1559 or EIP-2930 standards,
 //and Substrate API), as well as calculating the total fees in the block.
 
-// Define static basee fee per network
+// Define static base fee per network
```

Comment:
getting the same error in this file on like 58 that cannot convert to bigint, I think `refTime` needs to be added here to